life_expectancy/tests/test_load_save_data.py

```diff
@@ -0,0 +1,24 @@
+"""Test for load_data function"""
+from unittest.mock import patch
+import pandas as pd
+from life_expectancy.clean_data.utils.load_save_data import (
+    load_data, save_data
+)
+from . import FIXTURES_DIR
+
+
+def test_load_data():
+    """Test the data loading"""
+    df_input = load_data(FIXTURES_DIR / "eu_life_expectancy_raw.tsv")
+    assert df_input.shape == (100, 63)
+
+
+@patch("pandas.DataFrame.to_csv")
+def test_save_data(patched_to_csv):
+    """Test the data saving"""
+    patched_to_csv.return_value = "Hello!"
```

Comment:
Yeah, you are right! I thought I had to have some return value in order for it to work – removed in commit [d388ffe](https://github.com/2010b9/learning-pod-foundations-assignments/pull/6/commits/d388ffee2372728851815c842fea5f0509b4d3e0)
In that commit I also removed the `return df_to_save.to_csv(output_data_path, index=False)` (it's harmless, because it saves the `DataFrame` and returns `None`, but still I agree with you that it is odd.